### PR TITLE
Homebrew instructions

### PR DESCRIPTION
**Pull request by vurple**
Updated getting-started.asciidoc to include instructions for installing elasticsearch with Homebrew.

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md)?
- If submitting code, have you built your formula locally prior to submission with `gradle check`?
- If submitting code, is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
- If submitting code, have you checked that your submission is for an [OS that we support](https://www.elastic.co/support/matrix#show_os)?
- If you are submitting this code for a class then read our [policy](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md#contributing-as-part-of-a-class) for that.

### CODE REVIEW

**Comment by cbuescher** [comment]
Hi @vurple, 
thanks for opening this pull request. I'm not sure we want to add installation via Homebrew to the official docs since it is not one of out officially supported ways of installation.
I'll leave this open for a bit to leave room for others to comment though.

**Comment by jasontedor** [comment]
It's not officially supported as macOS is not officially supported, but it's important for our community that usage on macOS be simple and reliable which is why we work to ensure that Elasticsearch works on macOS on top of the OS that we do officially support (i.e., it's unofficially supported). So: we worked to ensure that installation could proceed [when Elasticsearch 5.0.0 was released](https://github.com/Homebrew/homebrew-core/pull/6301), and [we keep the formula up to date on each new release of Elasticsearch](https://github.com/Homebrew/homebrew-core/pulls?utf8=✓&q=is%3Apr%20is%3Aclosed%20author%3Ajasontedor%20elasticsearch%20). I think that including something here is okay, but I don't think it needs to be full instructions. We can simply say something like "Elasticsearch can be installed on macOS via Homebrew".

**Comment by jasontedor** [comment]
@vurple Can you update per my suggestion?

File: docs/reference/getting-started.asciidoc [hunk 1]
```diff
@@ -147,6 +147,19 @@ And now we are ready to start our node and single cluster:
 ./elasticsearch
 --------------------------------------------------
 
+=== Installation example with Homebrew
+
+--------------------------------------------------
+brew install elasticsearch
```
Comment (jasontedor) [inline docs/reference/getting-started.asciidoc]: I don’t think we need full details on how to use Homebrew here, a comment saying that Elasticsearch can be installed via Honebrew is sufficient.

**Review by jasontedor — CHANGES_REQUESTED** [review]
I left a formal review.

**Review by jasontedor — APPROVED** [review]
I pushed a few changes (fixed the heading, added some formatting, added a link to Homebrew, etc.) but this otherwise LGTM.

**Comment by jasontedor** [comment]
Thanks @vurple.

**Pull request merged**